Source/Engine.vcxproj

```diff
@@ -104,7 +104,7 @@
     <ClCompile>
       <WarningLevel>Level3</WarningLevel>
       <SDLCheck>false</SDLCheck>
-      <PreprocessorDefinitions>_CRT_SECURE_NO_WARNINGS;NOMINMAX;_SILENCE_CXX17_OLD_ALLOCATOR_MEMBERS_DEPRECATION_WARNING;_DEBUG;_CONSOLE;%(PreprocessorDefinitions);LOGGING_SUPPORT_DISABLED</PreprocessorDefinitions>
+      <PreprocessorDefinitions>_CRT_SECURE_NO_WARNINGS;NOMINMAX;_SILENCE_CXX17_OLD_ALLOCATOR_MEMBERS_DEPRECATION_WARNING;_DEBUG;_CONSOLE;%(PreprocessorDefinitions);LOGGING_SUPPORT_DISABLED;PLAY_BUILD</PreprocessorDefinitions>
```

Comment:
This change is intended to do? Just asking but I assume this is for the rush

Reply:
This is something related with play mode, its used to build the game without the engine itself. Miquel worked on it and I just merge it.

Reply:
I understand. My question is because merging this into development branch, will always apply this configuration into Debug build

Reply:
yes. I remove it for the merge

Reply:
Ok!